core-tests/src/test/java/org/mule/runtime/core/api/processor/LoggerMessageProcessorTestCase.java

```diff
@@ -35,7 +37,7 @@
 
   @Before
   public void before() throws RegistrationException {
-    flow = builder("flow", mockContextWithServices()).build();
+    flow = builder("flow", mockContextWithServices()).messageProcessors(emptyList()).build();
```

Comment:
`messageProcessors` or `processors`, same with source?  Why is emptyList needed? Does builder not use empty list by default?